### PR TITLE
Statistical functions' docs refactoring

### PR DESCRIPTION
**Pull request by annstriganova**

### CODE REVIEW

File: rule-engine/functions-time.md [hunk 1]
```diff
@@ -110,6 +110,15 @@ The fields of the DateTime object can be accessed with getter methods:
...
 * `getMinuteOfDay()`
+* `getMinuteOfHour()`
+* `getMonthOfYear()`
+* `getSecondOfDay()`
+* `getSecondOfMinute()`
+* `getWeekOfWeekyear()`
+* `getWeekyear()`
+* `getYear()`
+* `getYearOfCentury()`
+* `getYearOfEra()`
```
Comment (rodionos) [inline rule-engine/functions-time.md]: Lets also print the values for these getters for a sample date, e.g. 2018-01-13T16:45:22.303Z. You can create a table with method name and value for the above date.

File: rule-engine/functions-statistical.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+## Interval Selection
+
+By default, the statistical functions calculate the result based on all samples stored in the window. The range of samples can be adjusted by passing an optional argument - specified as sample count `c` or interval `i` - in which case the function will calculate the result based on the most recent samples.
+
+```javascript
+avg([string i | integer c]) double
+```
+
+* `avg(5)` - Average value for the last 5 samples.
+* `avg('1 HOUR)` - Average value for the last 1 hour.
```
Comment (rodionos) [inline rule-engine/functions-statistical.md]: Missing single quote

File: rule-engine/functions-statistical.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+Same as `slope_per_second()/60`.
+
+### `slope_per_hour`
+
+```javascript
+  slope_per_hour() double 
+```
+Same as `slope_per_second()/3600`.
+
+### Conditional Functions
```
Comment (rodionos) [inline rule-engine/functions-statistical.md]: Section header must be at Level 2 `##`

File: rule-engine/functions-time.md [hunk 4]
```diff
@@ -110,6 +110,22 @@ The fields of the DateTime object can be accessed with getter methods:
...
+
+|**Method**| **Value** |
+|:---|:---|
+|date_parse('2018-01-13T16:45:22.303Z').getWeekyear()|2018|
+|date_parse('2018-01-13T16:45:22.303Z').getYear()|2018|
+|date_parse('2018-01-13T16:45:22.303Z').getYearOfCentury()|18|
+|date_parse('2018-01-13T16:45:22.303Z').getYearOfEra()|2018|
```
Comment (rodionos) [inline rule-engine/functions-time.md]: Complete the table for all getter methods. No need to repeat date_parse('2018-01-13T16:45:22.303Z'), just specify method name `getWeekyear()` in one column and its value in the other column.

File: rule-engine/functions-time.md [hunk 5]
```diff
@@ -120,12 +120,28 @@ The fields of the DateTime object can be accessed with getter methods:
...
 
+Examples for '2018-01-13T16:45:22.303Z':
+
 |**Method**| **Value** |
 |:---|:---|
-|date_parse('2018-01-13T16:45:22.303Z').getWeekyear()|2018|
-|date_parse('2018-01-13T16:45:22.303Z').getYear()|2018|
-|date_parse('2018-01-13T16:45:22.303Z').getYearOfCentury()|18|
-|date_parse('2018-01-13T16:45:22.303Z').getYearOfEra()|2018|
+|getCenturyOfEra|20|
```
Comment (annstriganova) [inline rule-engine/functions-time.md]: Looks strange, expected 21.

**Pull request merged**